CHANGES.txt

```diff
@@ -60,6 +60,11 @@ Breaking Changes
 Changes
 =======
 
+- Added support for ``INSERT INTO ... ON CONFLICT DO UPDATE SET col = val``
+  which is identical to ``INSERT INTO ... ON DUPLICATE KEY UPDATE col = val``.
```

Comment:
Do we deprecate the old syntax? If that is the case, we need another bullet point stating that.